# patch -p0 -i path.patch --dry-run
--- webpack.config.js	2024-07-07 15:39:08.957633400 +0900
+++ webpack.config_.js	2024-07-01 14:17:27.842272900 +0900
@@ -13,7 +13,8 @@
     devtool: 'cheap-module-source-map',
     output: {
         library: 'VirtualMachine',
-        filename: '[name].js'
+        filename: '[name].js',
+        publicPath: 'https://sohta02.sakura.ne.jp/tukurutch/'
     },
     module: {
         rules: [{
@@ -49,7 +50,8 @@
         },
         output: {
             libraryTarget: 'umd',
-            path: path.resolve('dist', 'web')
+            path: path.resolve('dist', 'web'),
+            publicPath: 'https://sohta02.sakura.ne.jp/tukurutch/'
         },
         module: {
             rules: base.module.rules.concat([
@@ -68,7 +70,8 @@
         },
         output: {
             libraryTarget: 'commonjs2',
-            path: path.resolve('dist', 'node')
+            path: path.resolve('dist', 'node'),
+            publicPath: 'https://sohta02.sakura.ne.jp/tukurutch/'
         },
         externals: {
             'decode-html': true,
@@ -91,7 +94,8 @@
         },
         output: {
             path: path.resolve(__dirname, 'playground'),
-            filename: '[name].js'
+            filename: '[name].js',
+            publicPath: 'https://sohta02.sakura.ne.jp/tukurutch/'
         },
         module: {
             rules: base.module.rules.concat([
--- src/engine/runtime.js	2024-07-07 15:39:08.957633400 +0900
+++ src/engine/runtime_.js	2024-06-30 23:22:03.595769500 +0900
@@ -1194,7 +1194,7 @@
             blockJSON[`message${outLineNum}`] = '%1';
             blockJSON[`args${outLineNum}`] = [{
                 type: 'field_image',
-                src: './static/blocks-media/repeat.svg', // TODO: use a constant or make this configurable?
+                src: 'https://sohta02.sakura.ne.jp/tukurutch/static/blocks-media/repeat.svg', // TODO: use a constant or make this configurable?
                 width: 24,
                 height: 24,
                 alt: '*', // TODO remove this since we don't use collapsed blocks in scratch
